src/blackdoor/cqbe/node/Node.java

```diff
@@ -28,7 +28,7 @@
 	private Updater updater;
 	private AddressTable addressTable;
 	private StorageController storageController;
-	private volatile int n = Address.ADDRESS_SIZE;
+	private volatile int n = Address.DEFAULT_ADDRESS_SIZE;
```

Comment:
Discard this change, you're going to induce your own merge conflict with your other pull.